theme.json

```diff
@@ -650,6 +650,9 @@
 					"lineHeight": "1.5"
 				}
 			},
+			"core/query-title": {
+				"css": "& span {font-style: italic;}"
```

Comment:
@carolinan what do you think about this?

Reply:
It is the only way to style the query object name (like category name). :)

Reply:
yeah, I just feel like it looks so much better like this